recipes/lomap2/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "lomap2" %}
+{% set version = "v2.0_alpha" %}
```

Comment:
This needs to target the `dev` label: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

Although it might be easier to just wait until the full version is released.

Reply:
I had that in this commit 366d72819d540ab7fb7d8f8db9089a9a188a06a1

But will that cause issues with the openfe package? Will conda-build be able to figure that out? 

Thanks so much for your review!

Reply:
Yes, that can be done: https://conda-forge.org/docs/maintainer/knowledge_base.html#creating-a-pre-release-build

But to be honest, it's probably simpler if you make a real release and use that. Since you are upstream as well, I would evaluate if that's possible.

Reply:
I've cut the full release of Lomap now, so we can target v2.0.0 proper